recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/doma.bb

```diff
@@ -56,4 +56,19 @@ do_install() {
         # Update GUEST_DEPENDENCIES by adding camerabe after sndbe
         sed -i 's/\<sndbe\>/& camerabe/' ${D}${sysconfdir}/init.d/guest_doma
     fi
+
```

Comment:
here as well, if support the override - we just need to implement the additional command like do_install_append_h3es3